install.sh

```diff
@@ -238,6 +238,20 @@ __install_zeek() {
                 "security:zeek" \
                 "https://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/Release.key"
                 ;;
+            Debian)
+                __install_packages gpg
```

Comment:
Please test running the installation script on Debian without Zeek by passing the flag `--disable-zeek`. 

Since setting up the repository for both MongoDB and Zeek requires trusting a GPG key, I'm thinking that the MongoDB installation might not run unless gpg is installed. 

If this is the case, please remove this line and add `gpg` to the line `    for pkg in curl coreutils lsb-release yum-utils;` in `__install_installer_deps`

Reply:
@Zalgo2462  Sorry for the delay. As you expected, running the install script with the "--disable-zeek" flag caused a failure. After making the suggested change, everything works as expected.